api/client/out.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	"io"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/term"
+)
+
+// OutStream is an output stream used by the DockerCli to write normal program
+// output.
+type OutStream struct {
```

Comment:
It's possible, but there is very little shared code. I think there are only 3 single-line functions that are the same, so we don't gain all that much.